apps/condo/pages/ticket/[id]/index.tsx

```diff
@@ -49,6 +53,36 @@ const TicketDescriptionField: React.FC<ITicketDescriptionFieldProps> = ({ title,
     )
 }
 
+interface ITicketFileListProps {
+    files?: TicketFile[]
+}
+
+const TicketFileList: React.FC<ITicketFileListProps> = ({ files }) => {
+    const intl = useIntl()
+    const FilesFieldLabel = intl.formatMessage({ id: 'pages.condo.ticket.field.Files' })
+    const uploadFiles = files.map(({ file }) => {
+        const fileInList = {
+            uid: file.id,
+            name: file.originalFilename,
+            status: 'done' as UploadFileStatus,
+            url: file.publicUrl,
+        }
+        return fileInList        
+    })
+    const FilesFieldValue = (
+        <div className={'upload-control-wrapper'}>
+            <UploadList locale={{}} showRemoveIcon={false} items={uploadFiles} />
```

Comment:
Why locale is empty

Reply:
This locale is used by the Upload react component for displaying errors. But here we are using an upload list only to display files, so we do not need to create an Upload component locale to pass it to UploadList. And this field is somehow required, so I passed empty object there